cocos/base/ccUtils.h

```diff
@@ -109,7 +109,42 @@ namespace utils
      * @return Returns an instance of sprite
      */
     Sprite* createSpriteFromBase64(const char* base64String);
+    
+    /**
+     * Search a node by name
+
+     * @return  Returns searched node or nullptr
+     */
+    Node*  searchNode(Node* levelRoot, const char* name, bool recursively = true);
```

Comment:
Yep, i mean this function should be recursively without parameter `recursively `.